apps/passport-server/src/services/telegramService.ts

```diff
@@ -193,6 +204,58 @@ export class TelegramService {
         this.rollbarService?.reportError(e);
       }
     });
+
+    // The "link <eventName>" command is a dev utility for associating the channel Id with a given event.
+    this.bot.command("link", async (ctx) => {
+      if (ctx.chat?.type === "private") {
+        await ctx.reply(
+          "To get you started, can you please add me as an admin to the telegram channel associated with your event? Once you are done, please ping me again with /setup in the channel."
+        );
+        return;
+      }
+
+      const admins = await ctx.getChatAdministrators();
+      const isAdmin = admins.some(
+        (admin) => admin.user.id === this.bot.botInfo.id
+      );
+      if (!isAdmin) {
+        await ctx.reply(
+          "Please add me as an admin to the telegram channel associated with your event."
+        );
+        return;
+      }
+
+      const channelId = ctx.chat.id;
+      const TEST_EVENT_NAME = ctx.match || "ProgCrypto (Internal Test)";
```

Comment:
can you explain why we use ctx.match, or what this variable is for?

Reply:
`ctx.match` is a command to get the next input from a user. So they can type `/link Devconnect` and we will get `Devconnect` as the name of the event to match. This will be changed in the Admin Bot PR coming soon, because users can select from a menu of possible events.

Reply:
I see. In that case, it makes more sense to have the variable name be `eventName` and then add a comment - something like
```
// TODO: Add menu of possible events
````

Reply:
Also, not sure if I like that it defaults to `"ProgCrypto (Internal Test)"`. Can we just have it throw an error if ctx.match is falsy?

Reply:
<img width="633" alt="CleanShot 2023-09-22 at 08 32 54@2x" src="https://github.com/proofcarryingdata/zupass/assets/42984734/00a34056-6151-405c-9fa2-dbcd39c29fb9">

gm @rrrliu, here's the new flow.